pkg/proxy/iptables/proxier_test.go

```diff
@@ -5852,83 +6938,594 @@ func TestEndpointSliceWithTerminatingEndpointsTrafficPolicyCluster(t *testing.T)
 			} else {
 				assertIPTablesRulesNotEqual(t, testcase.line, testcase.expectedIPTables, fp.iptablesData.String())
 			}
+			runPacketFlowTests(t, testcase.line, fp.iptablesData.String(), testNodeIP, []packetFlowTest{
+				{
+					name:     "pod to clusterIP after endpoints deleted",
+					sourceIP: "10.0.0.2",
+					destIP:   "172.30.1.1",
+					destPort: 80,
+					output:   "REJECT",
+				},
+				{
+					name:     "external to LB after endpoints deleted",
+					sourceIP: testExternalClient,
+					destIP:   "1.2.3.4",
+					destPort: 80,
+					output:   "REJECT",
+				},
+			})
 		})
 	}
 }
 
-func TestMasqueradeAll(t *testing.T) {
-	ipt := iptablestest.NewFake()
-	fp := NewFakeProxier(ipt)
-	fp.masqueradeAll = true
+func TestInternalExternalMasquerade(t *testing.T) {
```

Comment:
This is a great test.  I wish we could make setup even easier, but that's for later.